### PR TITLE
Completed unit-tests-jest

### PR DESCRIPTION
**Pull request by amorabryan**

### CODE REVIEW

File: unit-tests-jest/numbers.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { evenNumbers, toDollars, multiplyBy, divideBy } from './numbers';
+
```
Comment (RobertGardner) [inline unit-tests-jest/numbers.test.js:2]: This is a good set of happy path tests. You should also test the edge cases.

**Review by RobertGardner — APPROVED** [review]
